.github/workflows/test.yml

```diff
@@ -215,9 +215,8 @@ jobs:
       run: pip install -e '.[metal,webgpu,testing]' --extra-index-url https://download.pytorch.org/whl/cpu
     - name: Test LLaMA compile speed
       run: PYTHONPATH="." METAL=1 python test/external/external_test_speed_llama.py
-    #- name: Run dtype test
-    #  run: DEBUG=4 METAL=1 python -m pytest -n=auto test/test_dtype.py
-    # dtype test has issues on test_half_to_int8
+    - name: Run dtype test
+      run: DEBUG=4 METAL=1 python -m pytest -n=auto test/test_dtype.py
```

Comment:
also not all tests are included here, should consolidate the individual ones under `test/` to `python -m pytest -n=auto test/` like other backends

Reply:
Ah. Understand. I will update it. Thank you!